modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -121,6 +126,17 @@ class Configuration(object):
            For RSA keys, supported values are PKCS1v15, PSS.
            For ECDSA keys, supported values are fips-186-3, deterministic-rfc6979.
         """
+        if signature_max_validity is not None and signature_max_validity.total_seconds() < 0:
+            raise Exception("The signature max validity must be a positive value")
+        self.signature_max_validity = signature_max_validity
+        """The signature max validity, expressed as a datetime.timedelta value.
+        It must be a positive value.
+        """
+        if signed_headers is not None and '(expires)' in signed_headers:
```

Comment:
Can you also add the condition `self.signature_max_validity is None` into this if statement?
Then you can delete the inner if statement and just raise the exception.

Reply:
done.